examples/next-js-amp/pages/preview.js

```diff
@@ -0,0 +1,5 @@
+export const config = { amp: 'hybrid'};
+
+import page from './[[...path]]';
+
+export default page;
```

Comment:
Is this `/preview` route needed for previewing in the Builder editor?

Reply:
yeah it simulates amp and it's a lot closer to the end result